eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/trace/Trace.java

```diff
@@ -140,7 +141,7 @@ public Span addTraceInfoToSpan(Span span, CloudEvent cloudEvent) {
         }
 
         for (String entry : cloudEvent.getExtensionNames()) {
-            span.setAttribute(entry, cloudEvent.getExtension(entry) == null ? "" : cloudEvent.getExtension(entry).toString());
+            span.setAttribute(entry, cloudEvent.getExtension(entry) != null ? cloudEvent.getExtension(entry).toString() : "");
         }
```

Comment:
Now you see there is no problem with the original code here. The other one is in the same situation. 
In conclusion, this issue should not be posted. So let's wait the maintainer to make the final decision.